main.tf

```
@@ -19,11 +19,38 @@ resource "aws_api_gateway_rest_api" "this" {
   }
 }
 
+data "aws_iam_policy_document" "default" {
```

```suggestion
data "aws_iam_policy_document" "default" {
```
This module appears to use `this` instead of `default`